imports/ui/nav/navbar.jsx

```diff
@@ -95,7 +97,12 @@ class Navbar extends Component {
 		);
 	}
 }
+const NavbarContainer = withTracker(() => {
+	return {
+		username: Meteor.users.find({}).fetch().map(e => e.username),
```

Comment:
Although this will work, Meteor should be imported before use.
Fetching user can be done with just `Meteor.user()`.
This is a good start. We can merge it after these two changes are made. However I think the whole app can benefit from knowing user/employee info. So after this, you can create an issue to add the combined info(_id, first last name, username, emails, etc… of the current user) from user and employee into the auth route. And from there, you can insert the info into redux. So each time an auth required page is loaded, the user/employee info will update in redux for any component that may use it.

Reply:
so should I change this to `Meteor.user()` way and merge it. Then create an issue about the auth route?

Reply:
Yes please do so.

Reply:
I have changed it. However, dont know how to handle the case if Meteor.user() is undefined.